server/server.js

```diff
@@ -1,8 +1,25 @@
-var express = require('express');
-var app = express();
-var port = 1993;
-
-app.use(express.static(__dirname + '/../client/dist'));
-
-
-app.listen(port, () => console.log(`Listening on port:${port}!`))
\ No newline at end of file
+const express = require('express');
+
+const path = require('path');
+
+const app = express();
+
+const port = 3000;
+
+const Location = require('../db/db.js');
+
+app.use(express.static(path.join(__dirname, '/../client/dist')));
+
+app.get('/locations/:locationID/reviews', (req, res) => {
+  Location.findOne({ locationID: parseInt(req.params.locationID, 0) },
```

Comment:
Good call specifying the radix in the `parseInt` call to preserve browser compatibility. This is something I'll have to incorporate in my code when I use this function. Interestingly enough, MDN mentions that passing a 0 for the argument will yield the same behavior as passing either undefined or nothing at all for the second parameter. I wonder if legacy browser versions know how to handle this particular case.